src/main/java/org/dita/dost/reader/ChunkMapReader.java

```diff
@@ -232,7 +231,7 @@ private void chunkMap(final Element root) {
      * Create the new topic stump.
      */
     private void createTopicStump(final URI newFile) {
-        try (final OutputStream newFileWriter = new FileOutputStream(new File(newFile))) {
+        try (final OutputStream newFileWriter = job.getStore().getOutputStream(newFile)) {
```

Comment:
Chunking code still uses byte streams to write the temporary XML files. Until chunking code is refactored or replaces, the `Store` API needs to support `InputStream` and `OutputStream`.